File: Pr3.java
import java.util.Scanner;
public class Pr3 {
	public static void main(String[] args) {
	       
	       Scanner inp = new Scanner(System.in);
	       
        int n=153 , tn, temp, total = 0;
        

        
        tn = n;
        while (tn != 0)
        {
            temp = tn % 10;
            total = total + temp * temp * temp;
            tn /= 10;
        }

        if(total == n)
            System.out.println(n + " is an Armstrong number!");
        else
            System.out.println(n + " is not an Armstrong number!");
	}
}

File: Assignment_1_cpp/Pr3.cpp
#include <iostream>
using namespace std;

int main() 
{
    int n; 
    cout<<"Enter the number: ";
    cin>>n;
    int tn, temp, total = 0;

    tn = n;
    while (tn != 0) 
    {
        temp = tn % 10;
        total = total + temp * temp * temp;
        tn /= 10;
    }

    if (total == n)
        cout << n << " is an Armstrong number!" << endl;
    else
        cout << n << " is not an Armstrong number!" << endl;

}
